silverback/_cli.py

```diff
@@ -272,12 +272,28 @@ def new_workspace(
     else:
         raise click.UsageError("Must provide a name or a slug/name combo")
 
-    workspace = platform.create_workspace(
+    platform.create_workspace(
```

Comment:
It may be nice to display a name of what gets created

Reply:
There is a `click.echo` below that shows the workspace created 
`click.echo(f"{click.style('SUCCESS', fg='green')}: Created '{workspace_name}'")`.